lib/wash/exe/wash.js

```diff
@@ -551,7 +545,7 @@ Wash.prototype.dispatch = function(pathSpec, argv, redirPathSpec, redirMode) {
    * @param {!Path} path
    * @return {!Promise<StatResult>}
    */
-  var builtinExists = function(path) {
+  var findBuiltin = function(path) {
```

Comment:
I would make this function a member of `Wash.prototype`, next to `Wash.prototype.findExecutable()`: more symmetrical and would avoid recreation of the function on every command execution.

This is very very minor, though.